packages/bsky/src/api/com/atproto/admin/getRecord.ts

```diff
@@ -1,11 +1,10 @@
 import { InvalidRequestError } from '@atproto/xrpc-server'
 import { Server } from '../../../../lexicon'
 import AppContext from '../../../../context'
-import { adminVerifier } from '../../../auth'
 
 export default function (server: Server, ctx: AppContext) {
   server.com.atproto.admin.getRecord({
-    auth: adminVerifier(ctx.cfg.adminPassword),
+    auth: ctx.adminVerifier,
```

Comment:
we call this `triageVerifier` in the pds & `adminVerifier` here. maybe we sync up the terminology?

It looks like all `triageVerifier` does is also ensure that `credentials.triage` is true (which it always is if `credentials.valid` is true) so probably not a change in logic

I think I like the appview semantics better here 🤔 

altho i wish we had a word to differentiate between "admin as in basic auth triage/moderation/admin" & "admin as in basic auth with admin priveleges"

Reply:
Yup yup 👍 I should be able to work something sensible out there.